ml

```diff
@@ -103,59 +103,24 @@ def init(
     quiet=False,
     nomaps=False,
     lite=False,
+    deep_clean=False,
```

Comment:
is there a reason why we are keeping deep_clean here if it is deprecated?

Reply:
mainly for backwards compatibility, but I could see an argument to take it out completely